plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -620,7 +620,7 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
 
             RemoteTableName remoteTableName = new RemoteTableName(Optional.ofNullable(catalog), Optional.ofNullable(remoteSchema), remoteTargetTableName);
             String sql = createTableSql(remoteTableName, columnList.build(), tableMetadata);
-            execute(session, connection, sql);
+            execute(session, connection, sql, getExtraTableAndColumnComments(remoteTableName, session, connection, tableMetadata));
```

Comment:
Why you are not calling `setTableComment` by default here?

Reply:
I was thought add statement into `transaction`, yes, great suggestion. If we execute one by one, definitely calling the `setTableComment` is better.